crates/meroctl/src/cli/init.rs

```diff
@@ -35,7 +35,7 @@ pub struct InitCommand {
 
     /// Host to listen on for RPC
     #[clap(long, value_name = "HOST")]
-    #[clap(default_value = "127.0.0.1,::1")]
+    #[clap(default_value = "0.0.0.0,::1")]
```

Comment:
let's not listen on all interfaces by default, keep that explicit for the node operator

it's different with the swarm host since that needs to be as accessible as possible through all means for network communication